examples/inception/README.md

```diff
@@ -17,9 +17,9 @@ Unzip the file and set the unzipped directory path as environment variable `SNPE
 ### Configure SNPE
 ```sh
 # in general, python 3.8 is recommended
-python -m olive.platform_sdk.qualcomm.snpe.configure --py_version 3.8
+python -m olive.platform_sdk.qualcomm.configure --py_version 3.8
```

Comment:
This does not have the sdk provided but it is required https://github.com/microsoft/Olive/blob/main/olive/platform_sdk/qualcomm/configure.py#L97

Reply:
can you update here too? https://github.com/microsoft/Olive/blob/main/examples/vgg/README.md

Reply:
Thanks. Done.